BuildModule.cmake

```diff
@@ -0,0 +1,19 @@
+include (../../flags.cmake)
```

Comment:
Undocumented feature but it is preferred to replace this with `include (${build_environment}/flags.cmake)`